esp/mlrs-wireless-bridge/mlrs-wireless-bridge.ino

```diff
@@ -401,6 +412,8 @@ if (g_protocol == WIRELESS_PROTOCOL_UDPCl) {
     DBG_PRINTLN(WiFi.localIP());
 
     setup_wifipower();
+    ip_udp = WiFi.broadcastIP();  // For easy GS configuration:  Enabling UDP should be enough
```

Comment:
except that it promises to make something easy, I really don't understand the comment. What does this make easier than before?

Reply:
Before, it was required to setup a UDPCl connection in MP with our address specified (which one would have to obtain from or configure in their network DHCP server.  By allowing our end to send first and by using broadcast for the first message, we eliminate the need for the ground station to know our address ahead of time.